clang/lib/AST/PrintfFormatString.cpp

```diff
@@ -484,6 +484,26 @@ bool clang::analyze_format_string::parseFormatStringHasFormattingSpecifiers(
   return false;
 }
 
+ArgType clang::analyze_format_string::wToArgType(
+    int size, bool fast, ASTContext &C) {
+  ArgType fastType = C.getTargetInfo().getTriple().isArch64Bit() ? C.LongLongTy : C.IntTy;
```

Comment:
This also seems incorrect -- we should be looking at the underlying type for the specified fastest (or exact) width integer. But again, not entirely clear how to do that from this interface...